### ISSUE TITLE
[BUG][Win] SQL Errors during setup

### ISSUE THREAD

**Issue by DJPlaya** [WeebDev/chibisafe#301]
**Describe the bug**
This might be an false-positive report since i am not that experienced with Linux and the Software isent intended to run on Windows.

When using npm run setup with MySQL, some non-critical SQL Errors are spit out:

**To Reproduce**
1. Clone Repo
2. Install node on Win Server 2019
3. install pm2
4. npm i
5. npm run setup
6. select MySQL as Database

**Additional Information**
SQL Dump attached
[local_chibisafe.sql.txt](https://github.com/WeebDev/chibisafe/files/7325443/local_chibisafe.sql.txt)

The Server is running Version 8.0.18 of MySQL Community

Errors:
```
> chibisafe@4.0.2 migrate C:\Users\Administrator\Desktop\Servers\chibisafe-master
> knex migrate:latest

Error: ER_TRUNCATED_WRONG_VALUE_FOR_FIELD: Incorrect string value: '\xF0\x9F\x9A\x80"' for column 'value' at row 1
    at Query.Sequence._packetToError (C:\Users\Administrator\Desktop\Servers\chibisafe-master\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\Administrator\Desktop\Servers\chibisafe-master\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\Administrator\Desktop\Servers\chibisafe-master\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\Administrator\Desktop\Servers\chibisafe-master\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\Administrator\Desktop\Servers\chibisafe-master\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\Administrator\Desktop\Servers\chibisafe-master\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\Servers\chibisafe-master\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\Servers\chibisafe-master\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:400:28)
    at addChunk (internal/streams/readable.js:290:12)
    --------------------
    at Protocol._enqueue (C:\Users\Administrator\Desktop\Servers\chibisafe-master\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\Administrator\Desktop\Servers\chibisafe-master\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\Administrator\Desktop\Servers\chibisafe-master\node_modules\knex\lib\dialects\mysql\index.js:134:18
    at new Promise (<anonymous>)
    at Client_MySQL._query (C:\Users\Administrator\Desktop\Servers\chibisafe-master\node_modules\knex\lib\dialects\mysql\index.js:128:12)
    at Client_MySQL.query (C:\Users\Administrator\Desktop\Servers\chibisafe-master\node_modules\knex\lib\client.js:168:17)
    at C:\Users\Administrator\Desktop\Servers\chibisafe-master\node_modules\knex\lib\transaction.js:325:24
    at new Promise (<anonymous>)
    at Client_MySQL.trxClient.query (C:\Users\Administrator\Desktop\Servers\chibisafe-master\node_modules\knex\lib\transaction.js:320:12)
    at Runner.query (C:\Users\Administrator\Desktop\Servers\chibisafe-master\node_modules\knex\lib\runner.js:151:36) {
  code: 'ER_TRUNCATED_WRONG_VALUE_FOR_FIELD',
  errno: 1366,
  sqlMessage: `Incorrect string value: '\\xF0\\x9F\\x9A\\x80"' for column 'value' at row 1`,
  sqlState: 'HY000',
  index: 0,
  sql: 'insert into `settings` (`key`, `value`) values (\'metaDescription\', \'\\"Blazing fast file uploader and bunker written in node! 🚀\\"\')'
}
Batch 1 run: 4 migrations

> chibisafe@4.0.2 seed C:\Users\Administrator\Desktop\Servers\chibisafe-master
> knex seed:run

Error: ER_TRUNCATED_WRONG_VALUE_FOR_FIELD: Incorrect string value: '\xF0\x9F\x9A\x80"' for column 'value' at row 1
    at Query.Sequence._packetToError (C:\Users\Administrator\Desktop\Servers\chibisafe-master\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\Administrator\Desktop\Servers\chibisafe-master\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\Administrator\Desktop\Servers\chibisafe-master\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\Administrator\Desktop\Servers\chibisafe-master\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\Administrator\Desktop\Servers\chibisafe-master\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\Administrator\Desktop\Servers\chibisafe-master\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\Servers\chibisafe-master\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\Servers\chibisafe-master\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:400:28)
    at addChunk (internal/streams/readable.js:290:12)
    --------------------
    at Protocol._enqueue (C:\Users\Administrator\Desktop\Servers\chibisafe-master\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\Administrator\Desktop\Servers\chibisafe-master\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\Administrator\Desktop\Servers\chibisafe-master\node_modules\knex\lib\dialects\mysql\index.js:134:18
    at new Promise (<anonymous>)
    at Client_MySQL._query (C:\Users\Administrator\Desktop\Servers\chibisafe-master\node_modules\knex\lib\dialects\mysql\index.js:128:12)
    at Client_MySQL.query (C:\Users\Administrator\Desktop\Servers\chibisafe-master\node_modules\knex\lib\client.js:168:17)
    at Runner.query (C:\Users\Administrator\Desktop\Servers\chibisafe-master\node_modules\knex\lib\runner.js:151:36)
    at C:\Users\Administrator\Desktop\Servers\chibisafe-master\node_modules\knex\lib\runner.js:40:23
    at C:\Users\Administrator\Desktop\Servers\chibisafe-master\node_modules\knex\lib\runner.js:277:24
    at processTicksAndRejections (internal/process/task_queues.js:95:5) {
  code: 'ER_TRUNCATED_WRONG_VALUE_FOR_FIELD',
  errno: 1366,
  sqlMessage: `Incorrect string value: '\\xF0\\x9F\\x9A\\x80"' for column 'value' at row 1`,
  sqlState: 'HY000',
  index: 0,
  sql: 'insert into `settings` (`key`, `value`) values (\'metaDescription\', \'\\"Blazing fast file uploader and bunker written in node! 🚀\\"\')'
}
```

**Comment by Aareksio** [comment]
The 🚀 emoji cannot fit in your database. From the dump you provided we can read the `value` field should be set to `utf8mb4_general_ci` which should fit the emoji. Such error in this context usually means your connection is not supporting emoji characters - probably your database defaults to something smaller than utf8mb4.

```sql
CREATE TABLE `settings` (
  `key` varchar(255) COLLATE utf8mb4_general_ci DEFAULT NULL,
  `value` varchar(255) COLLATE utf8mb4_general_ci DEFAULT NULL
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
```

You should be able to fix the issue by adjusting the connection charset [here](https://github.com/WeebDev/chibisafe/blob/master/src/api/structures/Database.js#L11-L17) before it is fixed in the upstream.

**Comment by DJPlaya** [comment]
This should be closed, the Error dident appeared again after 4af6c00
